dart/lib/src/sentry_options.dart

```diff
@@ -302,6 +319,9 @@ typedef SentryLogger = void Function(
   StackTrace? stackTrace,
 });
 
+typedef TracesSamplerCallback = double? Function(
```

Comment:
wondering if at some point we'd need to make the return as `FutureOr<double?` as we had to do it with some other interfaces, the problem is, if we do it now, startTransaction should return a `Future` too

Reply:
keep it sync and make it async only if necessary in the future